lib/cadence/client.rb

```diff
@@ -65,9 +65,9 @@ def register_domain(name, description = nil)
       nil
     end
 
-    def signal_workflow(workflow, signal, workflow_id, run_id, input = nil)
+    def signal_workflow(domain, signal, workflow_id, run_id, input = nil)
```

Comment:
maybe we can introduce a new method called `signal_workflow_execution` to reflect what is called on the connection?

this way we can make the method signature cleaner without breaking change